### PR TITLE
Update grammar, links, deprecations, etc in various text files

### PR DESCRIPTION
**Pull request by Chew**
Critical fixes.

Will have to add more later due to a headache. Remind me weekly.

### CODE REVIEW

**Comment by Chew** [comment]
What the heck git

**Comment by Camotoy** [comment]
`git rebase` ?

**Comment by Chew** [comment]
WHAT THE HECK GIT

**Comment by 7man7LMYT** [comment]
daily reminder

**Comment by 7man7LMYT** [comment]
daily reminder 2 - 3

**Comment by 7man7LMYT** [comment]
daily reminder 4

**Comment by 7man7LMYT** [comment]
daily reminder 5

**Comment by 7man7LMYT** [comment]
daily reminder 6

**Comment by 7man7LMYT** [comment]
daily reminder 7

**Comment by 7man7LMYT** [comment]
daily reminder 8

**Comment by 7man7LMYT** [comment]
daily reminder 9

**Comment by YHDiamond** [comment]
daily reminder for tomorrow so 7man doesnt have to (10)

**Comment by 7man7LMYT** [comment]
diamond it doesn't count, i said mine 5 hours ago, not a day so it doesn't count
so it's still (9)

**Comment by Chew** [comment]
Nah he was covering you for tomorrow so you can take a break

**Comment by 7man7LMYT** [comment]
No I have my calendar marks specifically for doign this

**Comment by 7man7LMYT** [comment]
i will continue until tjhhis mis merged

**Comment by 7man7LMYT** [comment]
daily reminder 10

**Comment by 7man7LMYT** [comment]
daily reminder 11

**Comment by 7man7LMYT** [comment]
daily reminder 11

**Comment by 7man7LMYT** [comment]
GITHUB WHY DID YOI

**Comment by 7man7LMYT** [comment]
daily reminder 12

**Comment by 7man7LMYT** [comment]
daily reminder 13

**Comment by 7man7LMYT** [comment]
daily reminder 14 (yay)

**Comment by 7man7LMYT** [comment]
daily reminder 15

**Comment by 7man7LMYT** [comment]
daily reminder 16

**Comment by Chew** [comment]
Daily? It's weekly and you've already reminded me this week 🤔

**Comment by 7man7LMYT** [comment]
image
yOU eDitEED >:(

**Comment by Chew** [comment]
rory

**Comment by Chew** [comment]
image
Smh I edited IN the line that says weekly 😤

**Comment by 7man7LMYT** [comment]
you changes it smh i'm calling camo police

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The following things can't be fixed because of Bedrock limitations. They might b
 ## Compiling
 1. Clone the repo to your computer
 2. [Install Maven](https://maven.apache.org/install.html)
-3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This downloads all the needed submodules for Geyser and is a crucial step in this process.
+3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This command downloads all the needed submodules for Geyser and is a crucial step in this process.
 4. Run `mvn clean install` and locate to the `target` folder.
```
Comment (Hellohi3654) [inline README.md:55]: Maybe you should add that if your having issues compiling try adding -U into the build command
E.g mvn clean install -U

Also you could add that adding -T 2C where 2 is the number of threads or cores (idk which one) maven uses

**Comment by 7man7LMYT** [comment]
weekly reminder 1.7 (est.)

**Comment by Chew** [comment]
@7man7LMYT the pr is ready for merge tho

**Comment by 7man7LMYT** [comment]
you said to remind you weekly, so i am!!1

i'm just doin what u said ;-;

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Take a look [here](https://github.com/GeyserMC/Geyser/wiki#Setup) for how to set
 - Structure block UI
 
 ## What can't be fixed
-The following things can't be fixed because of Bedrock limitations. They might be fixable in the future, but not as of now.
+We can't fix the following things because of Bedrock's limitations. They might be fixable in the future, but not as of now.
```
Comment (byquanton) [inline README.md:43]: I would not write "we" because these things cannot be changed by any translation software

**Comment by 7man7LMYT** [comment]
uhh is it weekly reminder 2? or 3 i forgot

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+*Pull requests that may not get accepted:* Niche features that apply to a specific group, for example, integration with a specific plugin. For now, please create a separate plugin if possible.
 
 *Pull requests for Floodgate:* Anything that opens up information within the game for developers to use.
 
-*Pull requests for Geyser:* Anything that fixes compatibility between Java or Bedrock, or improves the quality of play for Bedrock players. The exception is wherever direct server access is required; in this case it may be better for Floodgate.
+*Pull requests for Geyser:* Anything that fixes compatibility between Java or Bedrock or improves the quality of play for Bedrock players. The exception is wherever direct server access is required; in this case, it may be better for Floodgate.
 
 
-We have some general style guides that should be applied throughout the code:
+We have some general style guides that you should apply throughout the code:
```
Comment (Camotoy) [inline CONTRIBUTING.md:11]: Seems silly.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -44,8 +44,8 @@ public class LongClassName {
 }
 ```
 
-Make sure to comment your code where possible.
+Make sure to add comments to your code where possible.
```
Comment (Camotoy) [inline CONTRIBUTING.md:47]: Seems silly.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -44,8 +44,8 @@ public class LongClassName {
...
 
-The nature of our software requires a lot of arrays and maps to be stored - where possible, use Fastutil's specialized maps. For example, if you're storing block state translations, use an `Int2IntMap`.
+Our software's nature requires a lot of arrays and maps to be stored - where possible, use Fastutil's specialized maps. For example, if you're storing block state translations, use an `Int2IntMap`.
```
Comment (Camotoy) [inline CONTRIBUTING.md:49]: Seems silly.

File: README.md (hunk 2 above)
Comment (Camotoy) [reply README.md:43]: +1

File: README.md [hunk 6]
```diff
@@ -14,9 +14,9 @@ Geyser is an open collaboration project by [CubeCraft Games](https://cubecraft.n
 
 ## What is Geyser?
 Geyser is a proxy, bridging the gap between Minecraft: Bedrock Edition and Minecraft: Java Edition servers.
-The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
+This project's ultimate goal is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
```
Comment (Camotoy) [inline README.md:17]: Seems silly.

Comment (notnotnotswipez) [reply README.md:17]: I wonder if it seems silly, im not sure perhaps I need another confirmation

Comment (Konicai) [reply README.md:17]: It's worse haha

File: CONTRIBUTING.md (hunk 3 above)
Comment (Chew) [reply CONTRIBUTING.md:11]: Well, who's doing the applying? :p

File: README.md (hunk 6 above)
Comment (Chew) [reply README.md:17]: How is it silly/worse lol. There's no difference and gets to the point quicker

File: README.md (hunk 1 above)
Comment (Chew) [reply README.md:55]: @Camotoy thoughts? Barely out of the scope of this PR

File: README.md (hunk 2 above)
Comment (Chew) [reply README.md:43]: Yes.. that's covered by the "because of Bedrock's limitations"

File: CONTRIBUTING.md (hunk 4 above)
Comment (Chew) [reply CONTRIBUTING.md:47]: I still don't understand your definition of "silly"

File: README.md (hunk 1 above)
Comment (Camotoy) [reply README.md:55]: I would agree that is out of the scope of this PR. It's not a bad idea, however.

File: README.md (hunk 2 above)
Comment (Camotoy) [reply README.md:43]: The current wording doesn't place the blame on us exclusively. The listed things cannot be fixed by anyone, nor by anyone except the Bedrock developers.

File: CONTRIBUTING.md (hunk 5 above)
Comment (Chew) [reply CONTRIBUTING.md:49]: Okay this one I can agree on lol

File: CONTRIBUTING.md (hunk 3 above)
Comment (Camotoy) [reply CONTRIBUTING.md:11]: Everyone, not just the reader.

File: CONTRIBUTING.md (hunk 5 above)
Comment (Chew) [reply CONTRIBUTING.md:49]:
```suggestion
The nature of our software requires a lot of arrays and maps to be stored - where possible, use Fastutil's specialized maps. For example, if you're storing block state translations, use an `Int2IntMap`.
```

File: CONTRIBUTING.md (hunk 4 above)
Comment (Camotoy) [reply CONTRIBUTING.md:47]: I feel like "commenting your code" is used more in programming spaces as opposed to "adding comments to your code." Commenting is an act of itself.

File: README.md (hunk 6 above)
Comment (7man7LMYT) [reply README.md:17]: yeah original sounds better

**Review by Konicai — APPROVED** [review]

**Review by 7man7LMYT — APPROVED** [review]

**Review by toinouH — APPROVED** [review]
Nice

**Comment by 7man7LMYT** [comment]
daily reminder 25 

weekly reminder 2.5 i think

**Comment by Chew** [comment]
What is going on here

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -1,11 +1,11 @@
 name: Feature request
-about: Suggest an idea for this project
+description: Suggest an idea for this project
```
Comment (7man7LMYT) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: 10/10 needs to be merged

**Review by 7man7LMYT — APPROVED** [review]
10/10 pls merge

File: CONTRIBUTING.md (hunk 3 above)
Comment (Chew) [reply CONTRIBUTING.md:11]: Well the reader is reading these guidelines, right?

File: README.md (hunk 6 above)
Comment (Chew) [reply README.md:17]:
```suggestion
The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
```

File: CONTRIBUTING.md (hunk 4 above)
Comment (Chew) [reply CONTRIBUTING.md:47]:
```suggestion
Make sure to comment your code where possible.
```

**Comment by 7man7LMYT** [comment]
> What is going on here

rory

File: README.md (hunk 2 above)
Comment (Chew) [reply README.md:43]: "The following things can only be fixed by Bedrock developers. They might be fixable in the future, but not as of now."

Comment (Camotoy) [reply README.md:43]: Let's not blame the Bedrock developers. 

"The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser." or something.

**Review by x9nico — APPROVED** [review]

**Comment by YHDiamond** [comment]
Luke told me he doesn't want to change our discord links on github to be our vanity URL because we aren't sure if it is permanent yet... (Just noting that this seems to be a free PR moment imo)

**Comment by Chew** [comment]
Dunno how long ago that was. But, it's permanent, see: https://github.com/discord/discord-open-source/pull/303

**Comment by YHDiamond** [comment]
He told me it about 2 weeks ago, that pr hasn't been merged yet and he said that the vanity url was just given to us while we wait for the PR to be merged, just mentioning it

**Comment by Camotoy** [comment]
> Just noting that this seems to be an I want a free PR moment imo
Overruled. I'll allow it.

On the vanity invite: yeah, I don't expect it to break. When the open source PR is merged, nothing's going to happen to the vanity invite. We just have it now because it's a pain on that repository to merge new PRs.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Camotoy) [reply CONTRIBUTING.md:11]: We should let them know that we're not just enforcing the guidelines to you, but to everyone! :)

Comment (Chew) [reply CONTRIBUTING.md:11]: But you is everyone!! I don't get it

File: README.md (hunk 2 above)
Comment (Chew) [reply README.md:43]:
```suggestion
The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (Camotoy) [reply CONTRIBUTING.md:11]: You is only the reader. Commenting code applies to everyone, even if they do not read this document.

Comment (Chew) [reply CONTRIBUTING.md:11]:
```suggestion
We have some general style guides that should be applied throughout the code:
```

**Review by Camotoy — APPROVED** [review]

**Pull request merged**